org.sonarlint.eclipse.core/pom.xml

```
@@ -160,7 +160,7 @@
                     <artifactItem>
                       <groupId>com.sonarsource.cpp</groupId>
                       <artifactId>sonar-cfamily-plugin</artifactId>
-                      <version>6.68.0.85760</version>
+                      <version>6.69.0.86956</version>
```

Yesterday late afternoon a new release came out https://sonarsource.atlassian.net/browse/SLE-1249


```suggestion
                      <version>6.70.0.87073</version>
```